src/goInstallTools.ts

```diff
@@ -115,9 +115,11 @@ function installTools(goVersion: SemVersion, missing?: string[]) {
 
 	outputChannel.appendLine(''); // Blank line for spacing.
 
+	let http_proxy = vscode.workspace.getConfiguration('http').get('proxy');
+	let https_proxy = http_proxy;  // default proxy for http & https
```

Comment:
@ibigbug I understand the use case you are trying to solve. My question was for the use case where a user might have set the env var for http_proxy, but not in settings. What happens then, with your code changes?

I tested your code in 2 scenarios in Windows, this is what I see for the `env` that gets passed to the child process running the `go get`

1. When `http_proxy` is set in env var but not in VS Code settings
![image](https://cloud.githubusercontent.com/assets/16890566/21280400/9450aa7a-c39a-11e6-8517-de9fc10b89d7.png)

2. When different values are set for `http_proxy` in env var and VS Code settings
![image](https://cloud.githubusercontent.com/assets/16890566/21280433/c618c2f4-c39a-11e6-8e40-0fdddabb8e0c.png)

You will have to do 2 changes I believe
1. Use upper case while setting the http_proxy variable
2. If there is no VS Code setting for http_proxy, then do not change the process.env